tests/test_onnx/test_head.py

```diff
@@ -224,3 +215,289 @@ def test_yolov3_head_get_bboxes():
     for i in range(len(onnx_outputs)):
         np.testing.assert_allclose(
             torch_outputs[i], onnx_outputs[i], rtol=1e-03, atol=1e-05)
+
+
+def fcos_config():
+    """FCOS Head Config."""
+    test_cfg = mmcv.Config(
+        dict(
+            deploy_nms_pre=1000,
```

Comment:
try to test whether `deploy_nms_pre` config is valid in the testing.

Reply:
All head unittest case make the `deploy_nms_pre` valid.